de.dlr.sc.virsat.model.extension.cef.ui/src/de/dlr/sc/virsat/model/extension/cef/ui/templates/DLRCEFStudyCommandHelper.java

```diff
@@ -42,6 +42,15 @@
  */
 public class DLRCEFStudyCommandHelper {
 	
+	public static final String POWER_PARAMETERS = "powerParameters";
+	public static final String MASS_PARAMETERS = "massParameters";
+	public static final String SYSTEM_PARAMETERS = "systemParameters";
+	public static final String EQUIPMENT_MASS_PARAMETERS = "EquipmentMassParameters";

```

Comment:
Shouldn't all parameter start with a small or a big letter?
What do you prefer @franzTobiasDLR ?

Reply:
Classes / CAs upper case, properties lower case